package.json

```diff
@@ -5,18 +5,19 @@
         "packages/*"
     ],
     "dependencies": {
+        "@iobroker/adapter-react-v5": "file:packages/adapter-react-v5",
         "@iobroker/dm-gui-components": "file:packages/dm-gui-components",
         "@iobroker/json-config": "file:packages/jsonConfig",
-        "@iobroker/adapter-react-v5": "file:packages/adapter-react-v5",
         "iobroker.admin": "file:packages/admin"
     },
     "devDependencies": {
         "@alcalzone/release-script": "^3.8.0",
         "@alcalzone/release-script-plugin-iobroker": "^3.7.2",
-        "@alcalzone/release-script-plugin-license": "^3.7.0",
         "@alcalzone/release-script-plugin-lerna": "^3.7.0",
-        "@iobroker/types": "^7.0.7",
+        "@alcalzone/release-script-plugin-license": "^3.7.0",
         "@iobroker/dm-utils": "^1.0.10",
+        "@iobroker/eslint-config": "^2.0.1",
```

Comment:
moved eslint config to the package root (was only present in adapter-react-v5), else the backend had no ESLint support at least for my setup.